test/com/wsscode/pathom3/connect/planner_test.cljc

```diff
@@ -336,6 +336,124 @@
                                           :key          :b,
                                           :query        '...}}}))))))
 
+
+(deftest compute-run-graph--statically-unreachable-nodes-not-created
+  (testing "Nodes with nested provides that are known to be statically unreachable given nested query requires are not created"
+    (is (= #:com.wsscode.pathom3.connect.planner
```

Comment:
to keep things standard, can you please swipe the position and put `(= GIVEN EXPECTED)` please?

Reply:
Will do